CONTRIBUTING.md

```diff
@@ -47,137 +105,154 @@ the [Software Carpentry introduction to git][link_swc_intro].
+      * `fix/<some-identifier>`: for bugfixes
+      * `enh/<feature-name>`: for new features
+      * `doc/<some-identifier>`: for documentation improvements.
+        You should name all your documentation branches with the prefix `doc/` or `docs/`
+        as that will preempt triggering the full battery of continuous integration tests.
+
+1. **Make the changes you've discussed, following the [fMRIPrep coding style guide](#fMRIPrep-coding-style-guide).**<br />
+  Try to keep the changes focused: it is generally easy to review changes that address one feature or bug at a time.
+  It can also be helpful to test your changes locally,
+  using a [fMRIPrep development environment][link_devel].
+  Once you are satisfied with your local changes, [add/commit/push them][link_add_commit_push]
+  to the branch on your forked repository.
+
+1. **Submit a [pull request][link_pullrequest].**<br />
+   A member of the development team will review your changes to confirm
+   that they can be merged into the main code base.<br />
+   Pull request titles should begin with a descriptive prefix
+   (for example, `ENH: Support for SB-reference in multi-band datasets`):  
+     * `ENH`: enhancements or new features ([example][enh_ex])
+     * `FIX`: bug fixes ([example][fix_ex])
+     * `TST`: new or updated tests ([example][tst_ex])
+     * `DOC`: new or updated documentation ([example][doc_ex])
+     * `STY`: style changes ([example][sty_ex])
+     * `REF`: refactoring existing code ([example][ref_ex])
+     * `CI`: updates to continous integration infrastructure ([example][ci_ex])
+     * `MAINT`: general maintenance ([example][maint_ex])
+     * For works-in-progress, add the `WIP` tag in addition to the descriptive prefix.
+       Pull-requests tagged with `WIP:` will not be merged until the tag is removed.
+
+1. **Have your PR reviewed by the developers team, and update your changes accordingly in your branch.**<br />
+   The reviewers will take special care in assisting you address their comments, as well as dealing with conflicts
+   and other tricky situations that could emerge from distributed development.
+   And if you don't make the requested changes, we might ask
+   [@bedevere-bot](https://github.com/search?q=commenter%3Abedevere-bot+soft+cushions) 
+   to [poke you with soft cushions!](https://youtu.be/XnS49c9KZw8?t=1m7s)
 
 ## fMRIPrep coding style guide
 
-Whenever possible, instances of Nodes and Workflows should use the same names
+Whenever possible, instances of Nipype `Node`s and `Workflow`s should use the same names
 as the variables they are assigned to.
 This makes it easier to relate the content of the working directory to the code
 that generated it when debugging.
 
 Workflow variables should end in `_wf` to indicate that they refer to Workflows
 and not Nodes.
-For instance, a workflow whose basename is `myworkflow` might be defined as
+For instance, a workflow whose base-name is `myworkflow` might be defined as
```

Comment:
This doesn't feel more meaningful than "basename". You could add a space if it's important to you, but a hyphen is strange.

```suggestion
For instance, a workflow whose basename is `myworkflow` might be defined as
```